requirements/testing.txt

```diff
@@ -2,8 +2,10 @@
 -r local-docs-build.txt
 
 django-dynamic-fixture==2.0.0
-pytest==4.2.0
-pytest-django==3.4.7
+# pytest 4.2 is incompatible with pytest-django 3.4.6
+pytest==4.1.1  # pyup: <4.2
+# pytest-django 3.4.7 break our tests
+pytest-django==3.4.6 # pyup: <3.4.7
```

Comment:
This probably needs to say where is the problem, since upgrading these packages won't produce any error in the community site and we will upgrade them again.

Reply:
Test fail in the symlinks file if I remember correctly, but the stacktrace is really verbose, it can't even be displayed in my terminal.

I think I researched a little more and the problem was related to overriding the settings from django and not being restarted between some tests.

Reply:
So, I think the problem wasn't in that file, but when other tests run before that, still tricky to replicate

Reply:
The only way I can think is linking to this PR

Reply:
No need to link to this PR in the comment, add the explanation about _why_ this package can't be upgraded without checking that corporate tests are passing first.